layouts/default.vue

```diff
@@ -20,13 +23,15 @@ export default {
 }
 </script>
 <style>
-html {
-  padding: 0;
-  margin: 0;
-}
-.globe-component {
-  position: fixed;
-  z-index: -1;
-  top:0;
-}
+  /* normalize (reset to a common default) the stylesheets */
+  @import '~normalize.css/normalize.css';
```

Comment:
we can also add this to the nuxt config in 

```module.exports = {
  // Css entry file
  css: ['normalize.css/normalize.css'],```
https://nuxtjs.org/api/configuration-css

The person who named that package should take a hard look at themselves though...